portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+
+
+@WebServlet("/welcome")
+public class NewServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+
+    ArrayList<String> quotes = new ArrayList<String>();
+    quotes.add("Do not give up without trying");
+    quotes.add(" I love gaming");
+    quotes.add(" Respect is reciprocal");
+    quotes.add(" Thanks for checking out my portfolio");
+    
+
+    String json = convertToJsonUsingGson(quotes);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+    }
+    
+
+    private String convertToJsonUsingGson(ArrayList<String> quotes)
```

Comment:
From a code style perspective, I think that the opening curly brace is usually on the same line as the method signature.

Here is the relevant section of the Google Java style guide: https://google.github.io/styleguide/javaguide.html#s4.1-braces